### DISCUSSION TITLE
Struggling using carbon svelte as exported WebComponents.

### DISCUSSION THREAD

**Discussion by DZFLUX** [carbon-design-system/carbon-components-svelte#1238] [Q&A] [answered]
Hi everyone, I'm trying to use carbon-components-svelte as building blocks for some web components. I've cloned this repo to https://github.com/DayZeroLLC/carbon-components-svelte and made a couple svelte components that use some carbon-components-svelte.

~/lib/app.html has a html page trying to consume those components. I get errors testing with Accordion/AccordionItem/Breadcrumb/BreadcrumbItem. 

Any advice on what I'm doing incorrectly here?

Below is the console out compaining about flatpickr (when using AccordionItem) and my attempt to import it
```
<script src="https://cdnjs.cloudflare.com/ajax/libs/flatpickr/4.6.9/flatpickr.min.js" integrity="sha512-fWavsQbEkZyyE0Iiwx2ElW8jj95P3X/mDdUxiLWf4EyuW/AAah+fAbUBcBBIn2pOoGn4Y2+haVPK8VQJ7jw4Iw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

index.js:1 **Uncaught TypeError: Illegal constructor**
    at new Vt (index.js:1:9367)
    at new ge (index.js:1:21564)
    at Ev (index.js:1:685783)
    at Gt (index.js:1:9037)
    at new Nv (index.js:1:686624)
    at index.js:1:687182
    at index.js:1:173
    at index.js:1:273
Vt @ index.js:1
ge @ index.js:1
Ev @ index.js:1
Gt @ index.js:1
Nv @ index.js:1
(anonymous) @ index.js:1
(anonymous) @ index.js:1
(anonymous) @ index.js:1
app.html:1 Uncaught TypeError: **Failed to resolve module specifier "flatpickr"**. Relative references must start with either "/", "./", or "../".
app.html:1 Error handling response: TypeError: self.processResponse is not a function
    at chrome-extension://cmkdbmfndkfgebldhnkbfhlneefdaaip/js/notification.js:154:10
```

**Comment by metonym** [comment, marked as answer]
I haven't worked with using Svelte in conjunction with web components. To be honest, I don't think Carbon Svelte is viable for use as web components.

One of the biggest restrictions is that camel case props are not supported (e.g., `iconDescription`). Supporting web components has been proposed a couple of times: https://github.com/carbon-design-system/carbon-components-svelte/issues/467 https://github.com/carbon-design-system/carbon-components-svelte/issues/507

I would recommend looking at https://github.com/carbon-design-system/carbon-web-components for a native solution.

To anyone reading this thread: please chime in if you think otherwise.

**Reply by DZFLUX** [reply]
Thanks for the response Eric; We will use the web components.
